test/k8sT/assertionHelpers.go

```diff
@@ -111,7 +111,10 @@ func DeployCiliumAndDNS(vm *helpers.Kubectl) {
 	By("Installing DNS Deployment")
 	_ = vm.Apply(helpers.DNSDeployment())
 
-	DeployCiliumOptionsAndDNS(vm, []string{})
+	DeployCiliumOptionsAndDNS(vm, []string{
```

Comment:
Ouch, which values were used by default before?

Reply:
image `cilium`
tag `1.6.0`

Reply:
But `DeployCiliumOptionsAndDns` eventually calls https://github.com/cilium/cilium/blob/master/test/helpers/kubectl.go#L1041 which sets the following defaults:

```
                "global.registry":               "k8s1:5000/cilium",
		"agent.image":                   "cilium-dev",
		"agent.tag":                     "latest",
```

Reply:
You're right, I missed this, but there seems to be a problem with this anyway (my local ginkgo env tries to deploy Cilium with 1.6.0 tag, as does CI.

Reply:
Seems like `addIfNotOverwritten` may have a bug in it.

Reply:
I've just checked on my local ginkgo run:
```
  cilium-agent:
    Container ID:  docker://3643a7a48f43aed47a7e3189e5cb3b503490f988b156c300eb6caacdc4af1879
    Image:         k8s1:5000/cilium/cilium-dev:latest
```

Reply:
@brb were running on 1.6 branch? This is excerpt from Cilium daemonset from one of failed 1.6 builds:
```
"image": "k8s1:5000/cilium/cilium-dev:v1.6.0",
```
I also reproduced it in my local run (preflight check failed because there is no 1.6.0 tag of this image.

Pushed new version, seems like only tag was wrong because of wrong helm value being used.

Reply:
No, I have been running from my branch which was checked out from the `master`.

Reply:
Have we lost test coverage with this mistake?

Reply:
many 1.6 builds seem to have failed because of that